test/integration/group_vars/nxos.yaml

```diff
@@ -11,3 +11,4 @@ nxapi:
   #password: "{{ nxos_nxapi_pass | default('admin') }}"
   transport: nxapi
   use_ssl: no
+  port: 8080
```

Comment:
@rcarrillocruz Hi Ricardo.  Why was this changed to 8080?  The default port should be `80`.  This change causes our integration tests to start failing today.